src/query/SwisstopoIdentify.ts

```diff
@@ -1,21 +1,27 @@
 import Cartographic from 'cesium/Source/Core/Cartographic';
+import Cartesian3 from 'cesium/Source/Core/Cartesian3';
 import {radiansToLv95} from '../projection';
 
 const getIdentifyUrl = ({geom2056, lang, layers}) =>
-  `https://api3.geo.admin.ch/rest/services/all/MapServer/identify?geometry=${geom2056}&geometryFormat=geojson&geometryType=esriGeometryPoint&mapExtent=0,0,100,100&imageDisplay=100,100,100&lang=${lang}&layers=all:${layers}&limit=1&returnGeometry=true&sr=2056&tolerance=50`;
+  `https://api3.geo.admin.ch/rest/services/all/MapServer/identify?geometry=${geom2056}&geometryFormat=geojson&geometryType=esriGeometryPoint&mapExtent=0,0,100,100&imageDisplay=100,100,100&lang=${lang}&layers=all:${layers.join()}&limit=1&returnGeometry=true&sr=2056&tolerance=100`;
 const getPopupUrl = ({layerBodId, featureId, lang}) =>
   `https://api3.geo.admin.ch/rest/services/api/MapServer/${layerBodId}/${featureId}/htmlPopup?lang=${lang}`;
 
 
+export interface IdentifyResult {
+  layerBodId: string;
+  featureId: string;
+}
+
 export default class SwisstopoIdentify {
 
   /**
-   * @param {import ('cesium/Source/Core/Cartesian3.js').default} position
-   * @param {string} layers
+   * @param {Cartesian3} position
+   * @param {Array<string>} layers
```

Comment:
We usually write `string[]` but it is also fine like this.

Reply:
Found this way in other files of the project, just copied it. Let's harmonize this another time.